gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -515,6 +526,12 @@ public int read(ByteBuffer byteBuffer) throws IOException {
     } catch (InterruptedException e) {
       cancelCurrentRequest();
       throw new IOException(e);
+    } finally {
+      long endTime = System.currentTimeMillis();
```

Comment:
Will `finally` be called if there is a return (L 525) in the try block?

Reply:
Yes, it would execute.